transport/server.hh

```diff
@@ -136,6 +136,9 @@ private:
         uint64_t execute_requests;
         uint64_t batch_requests;
         uint64_t register_requests;
+
+        // errors
```

Comment:
This comment is pointless; it contains no information beyond the code.

Reply:
I agree. It would have been useful if there were a bunch of separate error counters, but if they are just one variable - a map - and it's called "errors", then it's rather obvious.

Reply:
Yes, it's a leftover from before the `map` was used, my mistake